docs/modules/migrate/pages/rolling-restart.adoc

```diff
@@ -0,0 +1,45 @@
+= Upgrade Cluster to {enterprise-product-name}
+:description: You can use an in-place rolling restart to move each member on your Open Source cluster to an Enterprise cluster.
```

Comment:
This wording is a bit weird. 

Maybe something like
```suggestion
:description: You can use an in-place rolling restart to change each member to upgrade your cluster from Open Source to Enterprise cluster.
```

Reply:
changed to:

You can upgrade your Open Source cluster to an Enterprise cluster using an in-place rolling restart to change each member in turn.

Reply:
Good.